mail/templates/applicant_letter/body.html

```diff
@@ -14,7 +15,7 @@
                   <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
                       <p style="margin: 0 0 10px;">
                         Sincerely,<br />
-                        <img src="{{ base_url }}/images/signature-mariah-rawding.png" alt="Signature for Mariah Rawding" /><br />
+                        <img src="{{ base_url }}{% static "images/signature-mariah-rawding.png" %}" alt="Signature for Mariah Rawding" /><br />
```

Comment:
I think we need to keep `/static` since this could become an absolute URL in production and so `base_url` would interfere with it

Reply:
Do you mean `"{{ base_url }}/static/images/.....`?